adagrams/game.py

```diff
@@ -1,13 +1,80 @@
 from random import randint
-
+letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
 def draw_letters():
-    pass
+    drawed_letters=[]
+    all_letters=[]
+    letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
+    for letter, quantity in letters_pool.items():
+        for i in range(quantity):
+            all_letters.append(letter)
+    while len(drawed_letters)<10:
+        random_number= randint(0,len(all_letters)-1) 
+        drawn_letter=all_letters[random_number]
+        drawed_letters.append(drawn_letter)
+        all_letters.pop(random_number)
+        letters_pool[drawn_letter] -= 1
+    return drawed_letters
 
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_count={}
+    word_count={}
+    word=word.upper()
+    for letter in letter_bank:
+        if letter not in letter_bank_count:
+            letter_bank_count[letter]=1
+        else:
+            letter_bank_count[letter]+=1
+
+    for letter in word:
+        if letter not in word_count:
+            word_count[letter]=1
+        else:
+            word_count[letter]+=1
+    for letter in word: 
+        if letter not in letter_bank or word_count[letter]>letter_bank_count[letter]:
+            return False
+ #   all checks passed then return True
+    return True
 
+#score_chart={['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T']:1,['D','G']:2,['B','C','M','P']:3,['F','H','V','W','Y']:4,['K']:5,['J','X']:8,['Q','Z']:10}
+score_chart={1:["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+             2:["D","G"],3:["B","C","M","P"],
+             4:["F","H","V","W","Y"],5:["K"],
+             8:["J","X"],10:["Q","Z"]}
```

Comment:
I'm left wondering what line 39. Should we use line 39 or 40 for `score_chart`? 

What happens if line 39 is accidentally uncommented? Then you would have code that shouldn't be executed that would be run. We should remove commented out code that isn't necessary. 

Also, `score_chart` is a constant variable so we should use all capital letters to name it.

Lastly, since the only place that `score_chart` is used is inside the `score_word` function, we should prefer to declare the variable inside the function so it doesn't have global scope.

```suggestion
SCORE_CHART = {1:["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
             2:["D","G"],3:["B","C","M","P"],
             4:["F","H","V","W","Y"],5:["K"],
             8:["J","X"],10:["Q","Z"]}
```